.gitignore

```diff
@@ -9,6 +9,7 @@
 *.swp
 .vimrc
 .kubeconfig
+.openshiftconfig
```

Comment:
There was a point where client looked in ./ for config. Handy for testing after compiling on my own machine instead of some slow cloud VM.